app/config/roles.php

```diff
@@ -8,6 +8,7 @@
     'home',
     'console',
     'graphql',
+    'sessions',
```

Comment:
Create a new `sessions` scope.

This scopes is available to guests, but only to app users (API keys) if created with the scope.